client/my-sites/stats/stats-module-devices/stats-module-devices.tsx

```diff
@@ -0,0 +1,125 @@
+import { SimplifiedSegmentedControl } from '@automattic/components';
+import { localizeUrl } from '@automattic/i18n-utils';
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import { useSelector } from 'calypso/state';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { JETPACK_SUPPORT_URL } from '../const';
+import useModuleDevicesQuery from '../hooks/use-modeule-devices-query';
+import StatsListCard from '../stats-list/stats-list-card';
+import StatsModulePlaceholder from '../stats-module/placeholder';
+
+// import '../stats-module/style.scss';
+// import '../stats-list/style.scss';
+import './stats-module-devices.scss';
+
+const OPTION_KEYS = {
+	SIZE: 'size',
```

Comment:
This is defined here to avoid using string as keys (to limit the risk for typos) and shouldn't be used for data query parameters.